backend-mock/lightning.js

```diff
@@ -77,9 +84,53 @@ router.get("/decode-pay-req", (req, res) => {
             },
           },
         ],
+        currency: "",
       })
     );
   }, 1500);
+
+  // CLN invoice
```

Comment:
What's this?

Reply:
Created an invoice with c-lightning to check if any fields differ.